controller/konnect/conditions.go

```diff
@@ -0,0 +1,42 @@
+package konnect
+
+// TODO(pmalek): move this to Konnect API directory so that it's part of the API contract.
```

Comment:
Can we open an issue in https://github.com/Kong/kubernetes-configuration to keep track of this?

Reply:
6955959821e472ccfb5f117c655a851f6fb23006